### DISCUSSION TITLE
Does KeepassXC support Yubikey selectively across devices?

### DISCUSSION THREAD

**Discussion by 1artist** [keepassxreboot/keepassxc#9726] [Q&A] [answered]
Is smart phone Android KeepassDX certified for accessing a KDBX file also used by my desktop Win10 KeepassXC app due to how Syncthing syncs the same KDBX file between the two devices? And since the Android KeepassDX app runs on a different OS, my concern is it can eventually corrupt my KDBX file preventing my desktop Win10 KeepassXC app from opening it? Is such a concern valid?

If the concern is not valid I need to know the following. I'm thinking of setting up Yubikey on the Android KeepassDX app to get easier access to the KDBX file on my smart phone. Does doing this mean my desktop Win10 KeepassXC app automatically also changes to requiring Yubikey to get access to the common KDBX file?  In short, is it possible for Yubikey access to only apply to my smart phone KeepassDX app without impacting my Desktop Win10 KeepassXC app access?  

In closing, I'm just seeking some clarity on what is the best approach to use a Syncthing'd common KDBX file between an Android smart phone and a Win10 desktop PC with Yubikey only used for access to the KDBX file while on my Android smart phone and not on my while on my desktop Win10 PC.  Thank you for your time.

**Comment by droidmonkey** [comment, marked as answer]
There is no concern between keepassdx and keepassxc, I use this combo every day.

Yes, if you add yubikey to kdbx then you must use it everywhere you use that kdbx.

**Reply by 1artist** [reply]
Thanks for the fast Droidmonkey. I will make sure to make a backup of my KDBX file before I start experimenting with such a setup. Do you by chance know of any instructional material that can guide me through setting up Yubikey on both my KeepassXC and KeepassDX?

**Reply by droidmonkey** [reply]
This should help: https://keepassxc.org/docs/#faq-yubikey-howto
